opm/material/fluidmatrixinteractions/EclTwoPhaseMaterial.hpp

```diff
@@ -125,6 +125,21 @@ class EclTwoPhaseMaterial : public TraitsT
      * \param params Parameters
      * \param state The fluid state
      */
+    template <class ContainerT, class FluidState>
```

Comment:
The comment block above describes the next method below (capillaryPressures), please move these functions above the comment block.